packages/builder/cypress/setup.js

```diff
@@ -29,22 +26,20 @@ process.env.ALLOW_DEV_AUTOMATIONS = 1
 // Stop info logs polluting test outputs
 process.env.LOG_LEVEL = "error"
 
-async function run() {
+exports.run = (
+  serverLoc = "../../server/dist",
```

Comment:
Just checking this doesn't actually fix the issue we had with needing to build, or does it?

Reply:
It does, there's a `ts/setup.ts` as you can't just run `ts-node`, you actually have to run it out of a typescript file for it to recognise it correctly, when ts-node is called it re-directs to the actual typescript server file instead.

Reply:
ah nice I missed that one good shout. Now that the worker TS has been merged can we update to include it here as well?